flixel/util/FlxBitmapData.hx

```diff
@@ -1,10 +1,25 @@
 package flixel.util;
 
-import flixel.util.FlxDestroyUtil.IFlxDestroyable;
+#if flash
+import flash.Vector;
+#else
+import openfl.Vector;
+#end
+
+import openfl.display.BlendMode;
+import openfl.display.StageQuality;
+import openfl.geom.Matrix;
+import openfl.utils.ByteArray;
 import openfl.display.BitmapData;
+import openfl.display.IBitmapDrawable;
 import openfl.display3D.textures.Texture;
+import openfl.filters.BitmapFilter;
+import openfl.geom.ColorTransform;
+import openfl.geom.Point;
+import openfl.geom.Rectangle;
 
 // TODO: implement it...
+// TODO: hack in openfl and lime asset management system to create FlxBitmapData objects (not just BitmapData)...
```

Comment:
the only reason i'm adding this class is for easier texture management support on flash target (maybe it would be better to have on openfl level, but i'm not sure).
i think i could force users to use this class. This will be another breaking change, so i think i'd better to include this class here as well.